src/functionalTest/resources/features/F-017/S-256.td.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title": "must successfully create single Role Assignment with only mandatory fields",
+  "_guid_": "S-256",
+  "_extends_": "F-017_Test_Data_Base_Get",
+  "specs": [
+    "has a actor id passed that does exists",
+    "contains a single Role Assignment with only mandatory fields",
+    "Befta1 - who invokes the API"
+  ],
+  "request": {
+    "pathVariables": {
+      "actorId": "${[scenarioContext][testData][users][invokingUser][id]}"
+    }
+  },
+  "expectedResponse": {
+    "responseCode" : "200",
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Length": "[[ANYTHING_PRESENT]]",
+      "Content-Type": "application/json"
+    },
+    "body": {
+      "roleAssignmentResponse": [{
+            "id" : "[[ANYTHING_PRESENT]]",
+            "actorIdType" : "IDAM",
+            "actorId" : "[[ANYTHING_PRESENT]]",
+            "roleType" : "ORGANISATION",
+            "roleName" : "Role1",
+            "classification" : "RESTRICTED",
+            "grantType" : "STANDARD",
+            "roleCategory" : "PROFESSIONAL",
+            "readOnly" : false,
+            "beginTime" : "[[ANYTHING_PRESENT]]",
+            "endTime" : "[[ANYTHING_PRESENT]]",
+            "created" : "[[ANYTHING_PRESENT]]",
+            "attributes" : {
+              "caseAccessGroupId" : "12345",
+              "substantive" : "Y",
+              "contractType" : "Salaried",
+              "primaryLocation" : "UK",
+              "jurisdiction" : "PRIVATELAW",
+              "caseType" : "12345"
+            },
+            "authorisations" : [ "Test123" ]
+          },
+         {
+          "id" : "[[ANYTHING_PRESENT]]",
```

Comment:
Seems this is a test data issue. Should not get duplicate role assignments.
Please check the data.